tests/fixtures/math_guy/basic_sly_data.hocon

```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2023-2025 Cognizant Digital Business, Evolutionary AI.
+# All Rights Reserved.
+# Issued under the Academic Public License.
+#
+# You can be released from the terms, and requirements of the Academic Public
+# License by purchasing a commercial license.
+# Purchase of a commercial license is mandatory for any use of the
+# neuro-san SDK Software in commercial settings.
+#
+# END COPYRIGHT
+
+# This file defines everything necessary for a data-driven test.
+{
+    # Describes what agent to test against.
+    "agent": "math_guy",
```

Comment:
Test case hocon that makes sure that math_guy agent works as expected on its own
passing back sly_data to the client.